pkg/deploy/api/helpers.go

```diff
@@ -17,3 +18,14 @@ func DeploymentToPodLogOptions(opts *DeploymentLogOptions) *kapi.PodLogOptions {
 		LimitBytes:   opts.LimitBytes,
 	}
 }
+
+// ScaleFromConfig builds a scale resource out of a deployment config.
+func ScaleFromConfig(dc *DeploymentConfig) *extensions.Scale {
+	return &extensions.Scale{
+		ObjectMeta: kapi.ObjectMeta{
+			Name:              dc.Name,
+			Namespace:         dc.Namespace,
+			CreationTimestamp: dc.CreationTimestamp,
```

Comment:
> And if we're going to have it, why is it useful to be the dc.CreationTimestamp?

Because this is a dc subresource and it's supposed to carry information about the main resource. Probably. Already upstream: https://github.com/kubernetes/kubernetes/blob/8c182c2713ea6e1b8ffff1da11e0d802cacd0bd8/pkg/apis/extensions/helpers.go#L75